apiserver/facades/client/charms/client_test.go

```diff
@@ -500,8 +491,32 @@ func (s *charmsMockSuite) TestCheckCharmPlacementWithHeterogeneous(c *gc.C) {
 	c.Assert(result.OneError(), gc.ErrorMatches, "charm can not be placed in a heterogeneous environment")
 }
 
-func (s *charmsMockSuite) api(c *gc.C) *charms.API {
-	api, err := charms.NewCharmsAPI(
+// NewCharmsAPI is only used for testing.
+func NewCharmsAPI(
```

Comment:
No need to export & comment is useless

Reply:
I think we should fix this eventually, Boy Scout rule. This was better than it was.